main/res/values/strings.xml

```diff
@@ -851,7 +851,7 @@
     <string name="contentstorage_selectfolder_dialog_choice_copy">Copy all content</string>
     <string name="contentstorage_selectfolder_dialog_choice_move">Move all content</string>
     <string name="contentstorage_selectfolder_dialog_copy_move_finished_title">Folder %s: Copy/Move done</string>
-    <string name="contentstorage_selectfolder_dialog_copy_move_finished_msg_html"><![CDATA[<p>The copy/move operation had the result: %s. %d file(s) and %d subdirectories were copied/moved.</p><p>Tap \'ok\' finalize the directory change.</p>]]></string>
+    <string name="contentstorage_selectfolder_dialog_copy_move_finished_msg_html"><![CDATA[<p>The copy/move operation had the result: %s. %d file(s) and %d folders were copied/moved.</p><p>Tap \'ok\' finalize the folder change.</p>]]></string>
```

Comment:
Please use positional parameter.

Reply:
And we need quantity strings here, don't we?

Reply:
And no reason for \<![CDATA[...]]>

Reply:
Someone else needs to handle this as I do not know, what needs to be done for these changes.

Reply:
Let's start with removing the CDATA part and positional parameters:

`<string name="contentstorage_selectfolder_dialog_copy_move_finished_msg_html">The copy/move operation had the result: %1$s. Affected files: %2$d, affected folders: %3$d.\n\nTap \'ok\' finalize the folder change.</string>`

Maybe we can avoid quantity string (actually called `plurals`) completely here by using that text.

Or maybe:

`<string name="contentstorage_selectfolder_dialog_copy_move_finished_msg_html">Result: %1$s\nAffected files: %2$d\nAffected folders: %3$d\n\nTap \'ok\' finalize the folder change.</string>`

Reply:
Can we do this in a separate PR? Code changes are needed in conjunction with CDATA removal. I am willing to do this, but let's please first merge and finish this PR

Reply:
(reason: code currently expects HTML to be returned in this string, as the key suffix "_html" suggests)

Reply:
I think we need a separate change (issue).
 - Text optimization - `finalize` sounds strange, as we can not abort anymore
 - Text optimization - no html formatting within translation string
 - keyword optimization - no `html` there
 - support for plurals
 
So I would only make the changes for the renaming to folder and make lint happy with positional parameters.

Reply:
As we have removed the cancel option meanwhile in the corresponding dialog, we can IMHO leave out `\n\nTap \'ok\' finalize the folder change.` completely. In other dialogues we don't explain that you have to press OK either and especially it gets confusing if there are two different translations for OK in one language 😉

Reply:
Don't bother too much over these particular lines. They need to change anyway soon in the context of saf ux redesign